### PR TITLE
fix default alpha beta in hardswish and add comment for them in hardsigmod

### PR DESCRIPTION
**Pull request by songqun**
Close https://github.com/Tencent/ncnn/issues/2826

### CODE REVIEW

File: src/layer/hardswish.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ HardSwish::HardSwish()
 
 int HardSwish::load_param(const ParamDict& pd)
 {
-    alpha = pd.get(0, 0.2f);
+    alpha = pd.get(0, 1.f/6);
```
Comment (songqun) [inline src/layer/hardswish.cpp:27]: not sure whether we need 0.1666667f instead.

**Comment by songqun** [comment]
Hi @nihui, could you help review the MR...

File: src/layer/hardswish.cpp [hunk 2]
```diff
@@ -24,7 +24,7 @@ HardSwish::HardSwish()
 
 int HardSwish::load_param(const ParamDict& pd)
 {
-    alpha = pd.get(0, 0.2f);
+    alpha = pd.get(0, 1.f / 6);
```
Comment (nihui) [inline src/layer/hardswish.cpp:27]: This breaks backward compatibility.
Suppose there is one old model file using the default alpha value. You will get different output with or w/o this change.

**Review by nihui — CHANGES_REQUESTED** [review]

File: src/layer/hardswish.cpp (hunk 2 above)
Comment (songqun) [reply src/layer/hardswish.cpp]: Have reverted it and add some comments instead.

**Review by nihui — APPROVED** [review]

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**